packages/prosemirror-lwdita/src/commands.ts

```diff
@@ -177,6 +178,223 @@ export class InputContainer {
   }
 }
 
+/**
+ * Render an image upload dialog. with an overlay
```

Comment:
Small typo: Remove the dot after dialog, please.

Reply:
Removed it